pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/generators/ModelGenerator.java

```diff
@@ -255,6 +255,37 @@ public void generate(final Protobuf3Parser.MessageDefContext msgDef,
 			).replaceAll("\n","\n"+FIELD_INDENT);
 		}
 
+		// Add here hashCode() for object with a single int field.
+		if (fields.size() == 1) {
+			FieldType fieldType = fields.get(0).type();
+			switch (fieldType) {
+				case ENUM, INT32, UINT32, SINT32, FIXED32, SFIXED32 -> {
```

Comment:
I don't think we need this for enums. We do need to add it for 64bit integers as well though.

Reply:
Addressed.